### ISSUE TITLE
Allow theme developers to incorporate our generated breadcrumb as output

### ISSUE THREAD

**Issue by lengthofrope** [nextbuzz/buzz-seo#37]
We can add a filter for this.

**Comment by robdekort** [comment]
Yes please, it would be awesome if the css classes added to the wrapper, list, list items and (current) links are customizable would be awesome (BEM). Perhaps as arguments for the filter?
